packages/firebase/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@next-auth/firebase-adapter",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "version": "0.0.0",
+  "description": "Firebase adapter for next-auth.",
+  "keywords": [
+    "next-auth",
+    "next.js",
+    "firebase"
+  ],
+  "homepage": "https://next-auth.js.org",
+  "bugs": {
+    "url": "https://github.com/nextauthjs/next-auth/issues"
+  },
+  "main": "dist/index.js",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc"
+  },
+  "author": "Ron Houben",
+  "contributors": [
+    "Nico Domino <yo@ndo.dev>"
+  ],
+  "license": "MIT",
+  "peerDependencies": {
+    "next-auth": "^3.1.0"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.20",
+    "@types/next-auth": "^3.1.23",
+    "firebase-admin": "^9.4.2",
+    "jest": "^26.6.3",
+    "next-auth": "^3.1.0",
+    "ts-jest": "^26.5.0",
+    "typescript": "^4.1.3"
```

```suggestion
  "peerDependencies": {
    "next-auth": "^3.1.0",
    "firebase-admin": "^9.4.2"
  },
  "devDependencies": {
    "firebase-admin": "^9.4.2"
```

Most of these dependencies are installed globally, so not needed here.